keylime/tenant.py

```diff
@@ -201,6 +201,16 @@ def init_add(self, args):
         al_data = None
         if "allowlist" in args and args["allowlist"] is not None:
 
+            _policy_pcrs = list(self.tpm_policy.keys())
```

Comment:
We don't need to prefix method/function variables with underscore. That convention in python is just for package level things so they aren't exported.